.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -394,7 +426,7 @@ jobs:
 
       #only install x32 support libraries when to run x86_32 cases
       - name: install x32 support libraries
-        if: matrix.test_option == '$DEFAULT_TEST_OPTIONS' || matrix.test_option == '$THREADS_TEST_OPTIONS'
+        if: (matrix.test_option == '$DEFAULT_TEST_OPTIONS' || matrix.test_option == '$THREADS_TEST_OPTIONS') && matrix.running_mode != 'fast-jit' && matrix.running_mode != 'jit'
```

Comment:
The line is too long, can we put `&& matrix.running_mode != 'fast-jit' && matrix.running_mode != 'jit'` to the next line?